<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>CertDetail</class>
 <widget class="QDialog" name="CertDetail" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="TextLabel1" >
       <property name="font" >
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text" >
        <string>Details of the Certificate</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="image" >
       <property name="minimumSize" >
        <size>
         <width>95</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>95</width>
         <height>40</height>
        </size>
       </property>
       <property name="scaledContents" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="Widget" >
     <widget class="QWidget" name="unnamed" >
      <attribute name="title" >
       <string>S&amp;tatus</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>8</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="frame" >
         <property name="frameShadow" >
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>8</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="2" column="0" >
           <widget class="QLabel" name="TextLabel6" >
            <property name="text" >
             <string>Private key</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="TextLabel4" >
            <property name="text" >
             <string>Signed by</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="TextLabel1_2_2" >
            <property name="text" >
             <string>Signature algorithm</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="ClickLabel" name="signCert" />
            </item>
            <item>
             <widget class="ClickLabel" name="trustState" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>13</hsizetype>
                <vsizetype>13</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="TextLabel2" >
            <property name="text" >
             <string>Internal name</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QLineEdit" name="sigAlgo" >
            <property name="palette" >
             <palette>
              <active>
               <color>
                <red>128</red>
                <green>128</green>
                <blue>128</blue>
               </color>
               <color>
                <red>192</red>
                <green>192</green>
                <blue>192</blue>
               </color>
               <color>
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
               <color>
                <red>223</red>
                <green>223</green>
                <blue>223</blue>
               </color>
               <color>
                <red>128</red>
                <green>128</green>
                <blue>128</blue>
               </color>
               <color>
                <red>160</red>
                <green>160</green>
                <blue>164</blue>
               </color>
               <color>
                <red>128</red>
                <green>128</green>
                <blue>128</blue>
               </color>
               <color>
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
               <color>
                <red>128</red>
                <green>128</green>
                <blue>128</blue>
               </color>
               <color>
                <red>192</red>
                <green>192</green>
                <blue>192</blue>
               </color>
               <color>
                <red>192</red>
                <green>192</green>
                <blue>192</blue>
               </color>
               <color>
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
               <color>
                <red>0</red>
                <green>0</green>
                <blue>128</blue>
               </color>
               <color>
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
               <color>
                <red>0</red>
                <green>0</green>
                <blue>255</blue>
               </color>
               <color>
                <red>255</red>
                <green>0</green>
                <blue>255</blue>
               </color>
               <color>
                <red>231</red>
                <green>231</green>
                <blue>231</blue>
               </color>
              </active>
              <inactive>
               <color>
                <red>128</red>
                <green>128</green>
                <blue>128</blue>
               </color>
               <color>
                <red>192</red>
                <green>192</green>
                <blue>192</blue>
               </color>
               <color>
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
               <color>
                <red>223</red>
                <green>223</green>
                <blue>223</blue>
               </color>
               <color>
                <red>128</red>
                <green>128</green>
                <blue>128</blue>
               </color>
               <color>
                <red>160</red>
                <green>160</green>
                <blue>164</blue>
               </color>
               <color>
                <red>128</red>
                <green>128</green>
                <blue>128</blue>
               </color>
               <color>
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
               <color>
                <red>128</red>
                <green>128</green>
                <blue>128</blue>
               </color>
               <color>
                <red>192</red>
                <green>192</green>
                <blue>192</blue>
               </color>
               <color>
                <red>192</red>
                <green>192</green>
                <blue>192</blue>
               </color>
               <color>
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
               <color>
                <red>0</red>
                <green>0</green>
                <blue>128</blue>
               </color>
               <color>
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
               <color>
                <red>0</red>
                <green>0</green>
                <blue>255</blue>
               </color>
               <color>
                <red>255</red>
                <green>0</green>
                <blue>255</blue>
               </color>
               <color>
                <red>231</red>
                <green>231</green>
                <blue>231</blue>
               </color>
              </inactive>
              <disabled>
               <color>
                <red>128</red>
                <green>128</green>
                <blue>128</blue>
               </color>
               <color>
                <red>192</red>
                <green>192</green>
                <blue>192</blue>
               </color>
               <color>
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
               <color>
                <red>223</red>
                <green>223</green>
                <blue>223</blue>
               </color>
               <color>
                <red>128</red>
                <green>128</green>
                <blue>128</blue>
               </color>
               <color>
                <red>160</red>
                <green>160</green>
                <blue>164</blue>
               </color>
               <color>
                <red>128</red>
                <green>128</green>
                <blue>128</blue>
               </color>
               <color>
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
               <color>
                <red>128</red>
                <green>128</green>
                <blue>128</blue>
               </color>
               <color>
                <red>192</red>
                <green>192</green>
                <blue>192</blue>
               </color>
               <color>
                <red>192</red>
                <green>192</green>
                <blue>192</blue>
               </color>
               <color>
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
               <color>
                <red>0</red>
                <green>0</green>
                <blue>128</blue>
               </color>
               <color>
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
               <color>
                <red>0</red>
                <green>0</green>
                <blue>255</blue>
               </color>
               <color>
                <red>255</red>
                <green>0</green>
                <blue>255</blue>
               </color>
               <color>
                <red>231</red>
                <green>231</green>
                <blue>231</blue>
               </color>
              </disabled>
             </palette>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="descr" >
            <property name="toolTip" >
             <string>The internal name of the certificate in the database</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="ClickLabel" name="privKey" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>13</hsizetype>
                <vsizetype>13</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="TextLabel1_3" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>0</hsizetype>
                <vsizetype>1</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Serial</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="serialNr" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>5</hsizetype>
                <vsizetype>1</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip" >
               <string>The serial number of the certificate</string>
              </property>
              <property name="frameShape" >
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow" >
               <enum>QFrame::Sunken</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>GroupBox</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>8</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="1" >
           <widget class="QLabel" name="fpSHA1" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>1</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip" >
             <string>A SHA-1 hashsum of the certificate</string>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Sunken</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLabel" name="fpMD5" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>1</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip" >
             <string>An md5 hashsum of the certificate</string>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Sunken</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="TextLabel2_2" >
            <property name="text" >
             <string>SHA1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="TextLabel1_2" >
            <property name="text" >
             <string>MD5</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Validity</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>8</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="notBefore" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>1</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip" >
             <string>The time since the certificate is valid</string>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Sunken</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="notAfter" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>1</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip" >
             <string>The time until the certificate is valid</string>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Sunken</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ClickLabel" name="dateValid" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="unnamed_1" >
      <attribute name="title" >
       <string>&amp;Subject</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>11</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="DistName" name="subject" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>&amp;Issuer</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>11</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="DistName" name="issuer" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="unnamed_2" >
      <attribute name="title" >
       <string>&amp;Extensions</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>11</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QTextEdit" name="v3extensions" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="but_ok" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>ClickLabel</class>
   <extends>QLabel</extends>
   <header>widgets/clicklabel.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>DistName</class>
   <extends></extends>
   <header>widgets/distname.h</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>but_ok</sender>
   <signal>clicked()</signal>
   <receiver>CertDetail</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>259</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel" >
     <x>319</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
